docs/modules/ROOT/pages/explanation/decisions/efk-openshift-logging.adoc

```diff
@@ -0,0 +1,43 @@
+= RedHat OpenShift Logging (EFK) Stack as Logging Solution
+
+== Problem
+
+We need to store user logs and allow users a way to visualize them.
+
+//Relevant requirements::
+//* tbd, links to requirement page(s)
```

Comment:
Are you going to do this? Otherwise I'd suggest removing the comment.

Reply:
I'll just remove it 😅